lib/progress_dialog.dart

```diff
@@ -216,6 +236,20 @@ class ProgressDialog {
                               ? Alignment.bottomRight
                               : Alignment.bottomCenter,
                         )
+                      : Container(),
+                  isCancelable
```

Comment:
You can check if the "onCancelClick" listener is null instead of this variable.

Reply:
Yes, I know but you can use `isCancelable` for different purposes for your future needs